### DISCUSSION TITLE
主题是否支持显示代码语法

### DISCUSSION THREAD

**Discussion by zwy5001** [YunYouJun/hexo-theme-starry#368] [Q&A 疑难杂症]
![image](https://user-images.githubusercontent.com/62434660/202996522-5a2106e2-21d9-451b-9563-e6b8dd7c88bf.png)
---
Hexo能否实现以下效果，在Valaxy上才能实现吗
---
![image](https://user-images.githubusercontent.com/62434660/202997118-0b8b4fb0-bbe6-4687-a439-94393cd37893.png)

**Comment by YunYouJun** [comment]
指的是右上角的标记语言 `php` 吗？

**Reply by zwy5001** [reply]
嗯，我查阅了Hexo的相关文档，还有Prism.js的一些资料:
---
> ![image](https://user-images.githubusercontent.com/62434660/203024948-d8df80bf-cce7-4d78-89de-b05816427528.png)
> ![image](https://user-images.githubusercontent.com/62434660/203025448-af2603a6-e27c-4968-8184-b140febf7e51.png)

**我就是想把这个语言类型给显示出来，但是我没有html或者css相关经验 (**

**Reply by YunYouJun** [reply]
是否考虑直接使用 Valaxy?
如果你确实需要，可以发起新的 Issue，hexo-theme-yun 可以后续添加该功能。
